src/test/pybind/test_rados.py

```diff
@@ -384,6 +384,9 @@ def test_obj_xattrs(self):
             stored_xattrs[key] = value
         eq(stored_xattrs, xattrs)
 
+    def test_get_name(self):
+        eq(self.ioctx.get_name(), 'test_pool')
```

Comment:
could add a test for `get_id()` or `get_pool_id()` using `Rados.pool_lookup()`.

Reply:
I will add a test for get_pool_id, thanks a lot.